extensions/elasticsearch/elasticsearch-5/src/main/java/com/hazelcast/jet/elastic/ElasticSinkBuilder.java

```diff
@@ -62,6 +62,8 @@
  */
 public final class ElasticSinkBuilder<T> implements Serializable {
 
+    private static final long serialVersionUID = 1L;
```

Comment:
This class actually shouldn't be `Serializable` (also version 6. and 7). 

I will send a separate PR with a fix.